genie-common/src/main/java/com/netflix/genie/common/dto/JobStatus.java

```diff
@@ -78,7 +89,7 @@ public static JobStatus parse(final String value) throws GeniePreconditionExcept
      * @return True if the job is still actively processing in some manner
      */
     public boolean isActive() {
-        return this == INIT || this == RUNNING;
```

Comment:
Enum should know that it is active or not during initialization. Create a constructor JobStatus(boolean active).